clients/src/main/resources/common/message/ElectPreferredLeadersResponse.json

```diff
@@ -22,7 +22,7 @@
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
     { "name": "ReplicaElectionResults", "type": "[]ReplicaElectionResult", "versions": "0+",
-      "about": "The error code, or 0 if there was no error.", "fields": [
+      "about": "The election results, or an empty array if the requester did not have permission.", "fields": [
```

Comment:
if the requester did not have permission => if the requester did not have permission when the request asks for all partitions (i.e. if TopicPartitions is null)